libspawner/src/win32/runner.cpp

```diff
@@ -159,8 +158,7 @@ bool runner::init_process_with_logon(const std::string &cmd, const char *wd) {
     WaitForSingleObject(main_job_object_access_mutex, INFINITE);
     set_allow_breakaway(false);
 
-    STARTUPINFOW siw;
-    ZeroMemory(&siw, sizeof(siw));
```

Comment:
I would actually prefer the reverse. add explicit `ZeroMemory` calls everywhere (and change `memset` above).
See e.g. https://blogs.msdn.microsoft.com/oldnewthing/20050628-07/?p=35183